### DISCUSSION TITLE
Picture stuttering

### DISCUSSION THREAD

**Discussion by JohnFante22** [UzixLS/zx-sizif-512#39] [General]
I recently upgraded my setup with a RetroTink 5X https://www.retrotink.com/product-page/5x-pro to get an optimal picture from my retro consoles. The RGB picture is great from all my other consoles but on my Zx-sizif (rev D) the picture stutters 2/3s up.

You can see it here:

https://www.dropbox.com/s/xxzn7fr6lpzugb8/Picture%20error%202.MOV?dl=0
﻿﻿﻿﻿﻿https://www.dropbox.com/s/rkiqvgaztpmdu73/Picture%20error%201.MOV?dl=0

I remember reading somewhere that this is hardware related. Is this so and can it be fixed?

Thank you in advance!

**Comment by UzixLS** [comment]
1. Try to switch to 128/+3/48 timings (Pentagon timings uses a non-standard 48Hz refresh rate which may cause compatibility issues with various equipment);
2. If you use RGB connection - try to use dedicated CSYNC signal instead of Composite_video+SYNC. For rev.E this is done by installing two jumpers on J6 as on image:
![изображение](https://github.com/UzixLS/zx-sizif-512/assets/1556813/5187378e-6b79-41d9-8121-2076bdb5a89d)
For older revisions you need to use properly-wired SCART cable: pin 20 on SCART terminal should be connected to pin 5 (which is csync) - not pin 4 (which is composite+sync) - on minidin.

**Comment by JohnFante22** [comment]
Thank you very much for your quick reply :-) ! Changing to 128/+3/48 timings did not remove the issue. It got a little better but not much.

Regarding the jumpers. Firstly I found out, when I opened the case, that it is a D1 revision. I may be looking the wrong place but I can not see the J6 jumper at the place you indicated: 

<img width="479" alt="Skærmbillede 2023-06-21 kl  20 35 53" src="https://github.com/UzixLS/zx-sizif-512/assets/107305437/69b6d59c-b50d-42f0-b070-e7e16fc04a70">

Are J6 placed differently on the D1 board than the D board? I did not build the board my self.

<img width="987" alt="Skærmbillede 2023-06-21 kl  20 38 01" src="https://github.com/UzixLS/zx-sizif-512/assets/107305437/57190b49-0ff5-4429-88e6-92e0ddf290fb">

**Reply by UzixLS** [reply]
These jumpers was added in rev.E. For older revisions you need to use properly-wired SCART cable: pin 20 on SCART terminal should be connected to pin 5 (which is csync) - not pin 4 (which is composite+sync) - on minidin.

**Reply by JohnFante22** [reply]
Ok. Thank you!

Is there a place you can buy the right (not dirty ... ) kind of cable? If I mess up the fix. My soldering skills are ... questionable :-)

**Reply by UzixLS** [reply]
Sorry, I don't know any such places (anyway none of them is working with RU). 
Personally I've modified megadrive2 rgb cable from aliexpress (but most time I'm still using composite video for its vibe :) )

**Comment by JohnFante22** [comment]
Ok. Will give it a go! 

The pin out on the socket is this, right?

![32xpin](https://github.com/UzixLS/zx-sizif-512/assets/107305437/3f1acac8-a2ef-42f6-9d04-6785428ee830)

So basically I have to move the wire on pin 4 to pin 5? Is there more than one wire attached to 4, BTW? if not I should be able ...

**Reply by UzixLS** [reply]
Yep, you have to rewire pin 4 to pin 5

**Reply by JohnFante22** [reply]
Just a quick update. rewireing pin 4 to 5 and changing my Retrotinks output resoluton to 480p did the trick. Thank you :-)

Now I just need to update to latest firmware. Always good with summer project ....

**Comment by LuckStilon** [comment]
@JohnFante22 Try adjusting the image with the C43 trimmer. It is not intended to adjust the RGB signal (it is dedicated for composite), but in practice settings of this trimmer affect the RGB signal (I don't know how and why). Its improper setting manifests like on Your film. It would be best if you adjust this trimmer with the composite cable and then switch back to the RGB cable. Use 65 menu in Retroleum DiagROM installed in Sizif.
